tools/firmware.toit

```diff
@@ -554,6 +555,69 @@ flash parsed/cli.Parsed -> none:
   finally:
     directory.rmdir --recursive tmp
 
+qemu_image_cmd -> cli.Command:
```

Comment:
Shouldn't this just be part of the extract command?

cli.OptionEnum "format" ["binary", "elf", "ubjson", "qemu"]

Reply:
done.